README.md

```diff
@@ -7,6 +7,7 @@ A terraform module to setup OpenVPN on GCP.
 ```hcl
 module "openvpn" {
   source     = "../modules/terraform-openvpn-gcp"
+  name       = var.name
```

Comment:
I believe the prefix variable accomplishes the same things here. Maybe changing it from `prefix` to `name` makes more sense tho.

Reply:
Yeah, but prefix is not an input. It is hardcorded in the module.

Reply:
Alright. Thanks for that